WooCommerce/Classes/POS/Presentation/Item Selector/ItemList.swift

```diff
@@ -3,18 +3,57 @@ import enum Yosemite.POSItem
 import struct Yosemite.POSVariableParentProduct
 
 /// Displays a list of POS items or placeholder card based on the given state.
-struct ItemList: View {
+struct ItemList<HeaderView: View>: View {
+    enum BaseItem {
+        case root
+        case parent(POSItem)
+    }
+
+    @Environment(\.floatingControlAreaSize) var floatingControlAreaSize: CGSize
```

Comment:
nit: this could be private?
```suggestion
    @Environment(\.floatingControlAreaSize) private var floatingControlAreaSize: CGSize
```